src/processing/dom/index.js

```diff
@@ -19,11 +19,12 @@ const JAVASCRIPT_PROTOCOL_REG_EX         = /^\s*javascript\s*:/i;
 const URL_ATTRS                          = ['href', 'src', 'action', 'manifest', 'data'];
 
 const URL_ATTR_TAGS = {
-    href:     ['a', 'link', 'image', 'area', 'base'],
-    src:      ['img', 'embed', 'script', 'source', 'video', 'audio', 'input', 'frame', 'iframe'],
-    action:   ['form'],
-    manifest: ['html'],
-    data:     ['object']
+    href:       ['a', 'link', 'image', 'area', 'base'],
+    src:        ['img', 'embed', 'script', 'source', 'video', 'audio', 'input', 'frame', 'iframe'],
+    action:     ['form'],
+    manifest:   ['html'],
+    data:       ['object'],
+    'xml:base': ['svg']
```

Comment:
It is wrong. The `xml:base` attribute is available on all SVG elements.
See https://www.w3.org/TR/SVG/struct.html#Core.attrib.
We need to process the `xml:base` attribute not only for `svg` tag.
Need to override for all SVG elements - `<rect>`, `<desc>`, etc.